static/css/devhub/new-landing/base.less

```diff
@@ -29,11 +30,20 @@
 @color-scheme-dark-background: #1d3770;
 @color-scheme-dark-items: #40d38d;
 
+// Buttons
+@color-button-default: #0568ba;
+@color-button-primary: #29c978;
+
```

Comment:
These two are already defined as `@color-scheme-medium-background` (#0568ba) and `@color-scheme-medium-items` (#29c978) - not sure if we should re-use them.